modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/persistence/ExtendedEventDto.java

```diff
@@ -64,44 +60,207 @@
   @Column(name = "organization", length = 128)
   private String organization;
 
+  /** Capture agent id */
+  @Column(name = "capture_agent_id", length = 128)
+  private String captureAgentId;
+
+  /** recording start date */
+  @Column(name = "start_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date startDate;
+
+  /** recording end date */
+  @Column(name = "end_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date endDate;
+
+  /** source */
+  @Column(name = "source")
+  private String source;
+
+  /** recording state */
+  @Column(name = "recording_state")
+  private String recordingState;
+
+  /** recording last heard */
+  @Column(name = "recording_last_heard")
+  private Long recordingLastHeard;
+
+  /** review status */
+  @Column(name = "review_status")
+  private String reviewStatus;
+
+  /** review date */
+  @Column(name = "review_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date reviewDate;
+
+  /** presenters */
+  @Column(name = "presenters")
+  private String presenters;
+
+  /** opt out */
+  @Column(name = "optout")
+  private Boolean optOut;
+
+  /** last modified origin */
+  @Column(name = "last_modified_origin")
+  private String lastModifiedOrigin;
+
+  /** last modified date */
+  @Column(name = "last_modified_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date lastModifiedDate;
+
+  /** capture agent properties */
+  @Column(name = "capture_agent_properties")
+  private String captureAgentProperties;
+
+  /** workflow properties */
+  @Column(name = "workflow_properties")
+  private String workflowProperties;
+
+  private String checksum;
```

Comment:
does this not need a column name?

Reply:
Most likely yes. I will add it and see if everything still works. I don't see a reason to not have a `@Column` annotation.